routing/cross_mwm_graph.cpp

```diff
@@ -209,6 +209,7 @@ CrossMwmGraph::MwmStatus CrossMwmGraph::GetMwmStatus(NumMwmId numMwmId, string c
   case MwmDataSource::MwmNotLoaded: return MwmStatus::NotLoaded;
   case MwmDataSource::SectionExists: return MwmStatus::SectionExists;
   case MwmDataSource::NoSection: return MwmStatus::NoSection;
+  default: return MwmStatus::NoSection;
```

Comment:
No, it will silence the error if a new switch case will be added. The code should not return in this case, but CRASH immediately, so we can fix the issue before releasing it to the public.